### PR TITLE
[NMS] Add APN resources config view

### LINKED ISSUE #3993: [NMS] Add APN resources config view to the gateway config component

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Add APN resources config view
Capture d’écran 2021-02-03 à 20 34 37

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js [hunk 1]
```diff
@@ -13,10 +13,6 @@
  * @flow strict-local
  * @format
  */
-import type {DataRows} from '../../components/DataGrid';
-import type {EnodebInfo} from '../../components/lte/EnodebUtils';
-import type {lte_gateway} from '@fbcnms/magma-api';
-
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js]: Leave the `type` imports at the top, the rest of the files have imports organized that way.

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js [hunk 2]
```diff
@@ -107,51 +106,9 @@ export default function GatewayConfig() {
...
-  function editRan() {
-    return (
-      <AddEditGatewayButton
-        title={'Edit'}
-        isLink={true}
-        editProps={{
-          editTable: 'ran',
-        }}
-      />
+      <AddEditGatewayButton title={'Edit'} isLink={true} editProps={editTab} />
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js:112]: Actually, is this `isLink` prop still doing anything? It looks like we could remove it

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js [hunk 3]
```diff
@@ -391,3 +368,40 @@ function GatewayRAN({gwInfo}: {gwInfo: lte_gateway}) {
...
+        {title: 'Name', field: 'name'},
+        {title: 'Resource ID', field: 'id'},
+        {title: 'VLAN ID', field: 'vlanId'},
+      ]}
+      options={{
+        actionsColumnIndex: -1,
+        pageSizeOptions: [5],
+        toolbar: false,
+        paging: false,
+      }}
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js]: Would this not display all the APN resources if there were more than 5?

**Review by andreilee — CHANGES_REQUESTED** [review]
Nit and a question

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js (hunk 1 above)
Comment (HannaFar) [reply nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js]: oops I did not see that I moved it

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js (hunk 2 above)
Comment (HannaFar) [reply nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js:112]: Yes it does, if it is set to false it is a button an not a link

File: nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js (hunk 3 above)
Comment (HannaFar) [reply nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js]: Oh you're right I need to set paging: true

**Review by andreilee — APPROVED** [review]

**Pull request merged**